packages/builders/src/node/build/webpack/config.ts

```diff
@@ -0,0 +1,141 @@
+import * as webpack from 'webpack';
+import { Configuration, ProgressPlugin } from 'webpack';
+
+import * as ts from 'typescript';
+import { dirname, resolve } from 'path';
+
+import { LicenseWebpackPlugin } from 'license-webpack-plugin';
+import CircularDependencyPlugin = require('circular-dependency-plugin');
+import ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+
+import { BuildNodeBuilderOptions } from '../node-build.builder';
+import * as nodeExternals from 'webpack-node-externals';
+
+export const OUT_FILENAME = 'main.js';
+
+export function getWebpackConfig(options: BuildNodeBuilderOptions) {
+  const webpackConfig: Configuration = {
+    entry: [options.main],
+    mode: options.optimization ? 'production' : 'development',
```

Comment:
I would suggest not using this.
using mode: 'production' enables things like uglify and module concatenation: https://webpack.js.org/concepts/mode/
Neither of which do anything for a Nodejs app aside from making it harder to debug a stack traces in a prod app.
And making it slower to compile

Reply:
I've disabled the options that we do not want. Because webpack aims to bundle all javascript, I think it's likely that most of their optimizations will be applicable to all javascript. However, with the nature of the mode, it's more geared towards browsers... For now, I think we will add configuration to disable the individual portions of the optimiztaions. What do you think?